test/ci-test-weather.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Let's talk about this when you have a time. My understanding is that this problem is orthogonal to the way we define the testing approach, in the sense that even with the current shunit-based approach we'd have the same problem if we want to validate everything in CI, don't we?